### ISSUE TITLE
Replace Eveyln address w/ SF Harrison address

### ISSUE THREAD

**Issue by pdehaan** [mozilla/legal-docs#1663]
https://github.com/mozilla/legal-docs/search?q=evelyn

... because we're not there anymore

```
Attn: Mozilla – Legal Notices
331 E. Evelyn Ave.,
Mountain View, CA 94041
Legal-notices@mozilla.com
```

Should probably be https://www.mozilla.org/contact/spaces/san-francisco/ address.

**Comment by peiying2** [comment]
Thanks for raising this issue! These are older docs thus not been updated for some time. Is there a way to update them in a more automated way, or it needs to be manually done?

**Comment by pdehaan** [comment]
I imagine it'd be a manual copy/paste for somebody to go through these and submit one big PR. The only tricky part would be if some of the addresses are localized.
And i guess the other issue is, it might not matter in some cases unless the sites that _use_ these legal docs are redeployed. Otherwise the addresses would still be old on the websites and only updated in the source files here. 🤷 

Disclaimer: I have no authority over anything. I was just pointing out the old address to a no-longer Mozilla office.

**Comment by peiying2** [comment]
Thanks for raising the issue. Maybe on PR for the same document. The actual address should be left in English. Some may not be easy to figure out depending on the script.

**Comment by flodolo** [comment]
@litkaemily @mafeld 
Thoughts on this? Does it need to be put into your queue of changes?

**Comment by flodolo** [comment]
There are 3 documents in the `en` folder with the old address.
* better_web_terms.md
* mobile_partner_website_branding_terms.md
* mobile_partner_website_prototype_agreement.md

In other cases, it's been updated for English but not for other locales. As the first link points out, there are 55 occurrences of the old address, so it's unclear what needs to be done (e.g. can they be updated without any other change?).

**Comment by mafeld** [comment]
I don't think any of those pages are live. 

better_web has been sunset. 
And the mobile branding terms are from Firefox OS days. So I don't think there's any need to update them. 

If you see them live anywhere, let me know so we can clean them up.

**Comment by peiying2** [comment]
> I don't think any of those pages are live.
> 
> better_web has been sunset. And the mobile branding terms are from Firefox OS days. So I don't think there's any need to update them.
> 
> If you see them live anywhere, let me know so we can clean them up.

Shall we remove/archive those no longer needed?

Looks like `website_tou.md` needs to be updated for the localized versions. Can we update them without updating version number?

**Comment by mafeld** [comment]
Yes and Yes. 
Let's archive the no longer needed ones. 
And we can update the localized versions without updating the version number.

**Comment by flodolo** [comment]
@mafeld 

There's one more occurrence in a file that lives in the root of the repository.
https://github.com/mozilla/legal-docs/blob/master/Firefox-Addons-Distribution-Agreement-20150522

BTW, what's the story behind those files? I assume they're not used anywhere.

I'm going to open a PR with changes in separate commits, in case it would be easy to undo some parts.

**Comment by mafeld** [comment]
The website for the Firefox Add-on Distribution Agreement is still live (https://extensionworkshop.com/documentation/publish/firefox-add-on-distribution-agreement/), but I don't believe it pulls from the legal repo anymore. 

So I think it's ok to archive the verison in the legal repo.

**Comment by flodolo** [comment]
Indeed, this one is also from 2015 and unformatted, so probably a good idea to remove.

**Comment by flodolo** [comment]
Fixed by #1668
